mirascope/core/base/_utils.py

```diff
@@ -57,6 +58,15 @@ def format_prompt_template(template: str, attrs: dict[str, Any]) -> str:
                 values[var] = "\n".join([str(item) for item in attr])
         else:
             values[var] = str(attr)
+    return values
+
+
+def format_template(template: str, attrs: dict[str, Any]) -> str:
```

Comment:
can you command search for the old name and find/replace? I believe there are a few places that use this method that haven't been updated (like the `__str__` method in `BasePrompt` for example

Reply:
I fixed only the unittest name.
All other methods had been renamed to this name because of PyCharm's refactoring functionality.